src/components/GridSection.tsx

```diff
@@ -0,0 +1,16 @@
+interface Props
+  extends React.HTMLProps<HTMLDivElement>,
+    React.PropsWithChildren {}
+
+export const GridSection = ({ children, className, ...rest }: Props) => {
+  return (
+    <div
+      className={`grid gap-8 md:grid-cols-2 p-4 md:p-0 ${className}`}
```

Comment:
if you're doing .trim() everywhere else you probably want to do it here too :)

Reply:
Yeah being consistent is best practice.